packages/collapsible/README.md

```diff
@@ -0,0 +1,182 @@
+# Collapsible
+
+`lion-collapsible` is a combination of a button (the trigger), a chunk of 'extra content', and an animation that is used to disclose the extra content. There are two slots available respectively; `invoker` to specify collapsible invoker and `content` for the extra content of the collapsible.
+
+```js script
+import { html } from 'lit-html';
+
+import './lion-collapsible.js';
+import './demo/my-collapsible.js';
+
+export default {
+  title: 'Others/Collapsible',
+};
+```
+
+```js preview-story
+export const main = () => html`
+  <lion-collapsible>
+    <button slot="invoker">More about cars</button>
+    <div slot="content">
+      Most definitions of cars say that they run primarily on roads, seat one to eight people, have
+      four tires, and mainly transport people rather than goods.
+    </div>
+  </lion-collapsible>
+`;
+```
+
+## Features
+
+- Use `opened` or `toggle()` to render default open
+- `invoker` slot can be custom template e.g. `lion-button`
+- Observe the state with the help of `@opened-changed` event
+- `show()` and `hide()` are helper methods to hide or show the content from outside
+
+## How to use
+
+### Installation
+
+```bash
+npm i --save @lion/collapsible
+```
+
+```js
+import { LionCollapsible } from '@lion/collapsible';
+// or
+import '@lion/collapsible/lion-collapsible.js';
+```
+
+### Usage
+
+```html
+<lion-collapsible>
+  <button slot="invoker">Invoker Text</button>
+  <div slot="content">
+    Extra content
+  </div>
+</lion-collapsible>
+```
+
+### Examples
+
+#### Default Expanded
+
+Add the `opened` attribute to keep the component default open.
+
+```js preview-story
+export const defaultExpanded = () => html`
+  <lion-collapsible opened>
+    <button slot="invoker">More about cars</button>
+    <div slot="content">
+      Most definitions of cars say that they run primarily on roads, seat one to eight people, have
+      four tires, and mainly transport people rather than goods.
+    </div>
+  </lion-collapsible>
+`;
+```
+
+#### Custom Invoker Template
+
+A custom template can be specified to the `invoker` slot. It can be any button or custom component which mimics the button behavior for better accessibility support. In the below example, `lion-button` is used as a collapsible invoker.
+
+```js preview-story
+export const customInvokerTemplate = () => html`
+  <lion-collapsible>
+    <lion-button slot="invoker">More about cars</lion-button>
```

Comment:
I don't think this is imported right? Maybe change it to <div> or <small> as an example or something, prevents having to depend on lion-button.

Reply:
I can add custom styling to the native button, just to demonstrate that invoker can be customizable. Same as `customAnimation`.

Reply:
Sounds good as well :)